apps/playwright/tests/auth/admin.spec.ts

```diff
@@ -3,14 +3,44 @@ import AUTH from "~/constants/auth";
 import { loginBySub } from "~/utils/auth";
 
 test.describe("Admin user", () => {
-  test("Can access restricted paths", async ({ appPage }) => {
+  const restrictedPathsChunk1 = AUTH.RESTRICTED_PATHS.ADMIN.slice(0, 4);
+  const restrictedPathsChunk2 = AUTH.RESTRICTED_PATHS.ADMIN.slice(4);
+
+  test("Can access restricted paths group 1", async ({ appPage }) => {
+    await loginBySub(appPage.appPage, "admin@test.com");
+    await Promise.all(
+      restrictedPathsChunk1.map(async (restrictedPath) => {
+        const context = appPage.page.context();
+        const page = await context.newPage();
+        await page.goto(restrictedPath);
+        await page.waitForURL(restrictedPath);
+        await expect(
+          page.getByRole("link", {
+            name: /dashboard/i,
+          }),
+        ).toBeVisible();
```

Comment:
I found negative assertions, checking something is hidden, unreliable off page load. Tests would perform contrary to expectation. Likely due to asserting something was hidden at a loading screen.
So, a positive assertion first then negative assertion.